packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -165,7 +170,7 @@ const Input: React.FC<WithAsProp<InputProps>> & FluentComponentStaticProps<Input
       inline,
       disabled,
       clearable,
-      hasIcon: !!icon || !!successIndicator || !!error,
+      hasIcon: !!icon || showSuccessIndicator || required || error,
```

Comment:
Can we please send separately: `hasIcon`, `showSuccessIndicator`, `reuired` and `hasError` and do this condition in the theme? 

That way would be easier to do changes in the theme without necessarily requiring code changes.

Reply:
This is just the input condition for set the slot for the icon, we would be passing the props to styles only for that?

Reply:
My point was instead of doing this decision here, why not doing that in the styles based on 4 booleans. In the end is this component's logic of theme logic? I would expect that some themes may show these differently, so if they have access to all of them, that decision can be made in the theme, otherwise it will be locked to component's logic @layershifter @jurokapsiar @miroslavstastny any other opinions?

Reply:
After offline discussion we decided not to change this as part of this PR, we will revisit this if needed to be changed in a separete PR